cpu/sam0_common/include/cpu_conf.h

```diff
@@ -22,12 +22,43 @@
 #define CPU_CONF_H
 
 #include "cpu_conf_common.h"
-#if defined(CPU_SAML1X)
-#include "vendor/sam23.h"
-#elif defined(CPU_SAMD5X)
-#include "vendor/samd5x.h"
-#else
-#include "vendor/sam0.h"
+
+/* Workaround redefinition of LITTLE_ENDIAN macro (part1) */
+#ifdef LITTLE_ENDIAN
+#define __TMP_LITTLE_ENDIAN     LITTLE_ENDIAN
+#undef LITTLE_ENDIAN
+#endif
```

Comment:
I think we can drop this dance - I don't see `LITTLE_ENDIAN` being defined anywhere in RIOT headers.

Reply:
You can give it a try, Murdock will tell us.

Reply:
I'll give it a try once Murdock confirmed I didn't forget anything in the first go-around.

Reply:
Just tested it locally and it turns out the `LITTLE_ENDIAN` define comes from newlib.
Storing the 'little endian state' is still unnecessary as it will always be set for sam0, but let's keep it like it was then.